setup.py

```diff
@@ -64,24 +64,19 @@ def read(*rnames):
         return f.read()
 
 
-Cwrapper = Feature(
-    "C wrapper",
-    standard=True,
-    headers=[os.path.join('src', 'zope', 'proxy', 'proxy.h')],
```

Comment:
I may be overthinking this.  I think all we need to do is define a `headers` variable that's either an empty list or this list, and then pass it to `setup(..., headers=headers, ext_modules=ext_modules, ...)`.

Reply:
That's what [greenlet does](https://github.com/python-greenlet/greenlet/blob/829b44f62b024a3f9b79453b1c0c6a2eb766c336/setup.py#L81). I'll do that here. Thanks!

Reply:
With `headers=` as a keyword argument, I get this path installed into my virtualenv:
`$VIRTALENV/include/site/python3.8/zope.proxy/proxy.h`, which is exactly the same path I got on `master` using the `Feature`.